src/adaXT/predict/predict.pyx

```diff
@@ -196,7 +196,7 @@ cdef class PredictClassification(Predict):
                                                  unique_classes=unique_classes)
                 return np.mean(predictions, axis=0, dtype=int)
 
-        predictions = parallel.async_map(default_predict, trees, X=X_new,
+        predictions = parallel.async_map(predict_default, trees, X=X_new,
                                          **kwargs)
         return np.array(np.apply_along_axis(mode, 0, predictions), dtype=DOUBLE)
```

Comment:
Same with the dtype here. Perhaps we want int for classification.

Reply:
Mhh not sure I follow. Here its actually also the wrong way around. If we have predict_proba=True it should be DOUBLE and if not it should be int. Or am I missing something?